DiscordIan/Model/OpenWeatherMap/OpenWeatherMapCurrentModel.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+using System.Xml.Serialization;
+using System.Collections.Generic;
+
+namespace DiscordIan.Model.OpenWeatherMap
+{
+    public class WeatherCurrent
+    {
+        [XmlRoot(ElementName = "coord")]
```

Comment:
Are these attributes necessary or can the XML parser suss out the appropriate names on serialization/deserialization?

Reply:
From what I'm seeing they are necessary: https://stackoverflow.com/questions/9334771/case-insensitive-xml-parser-in-c-sharp